### ISSUE TITLE
🚀 - Support for custom color for text

### ISSUE THREAD

**Issue by juraj-hrivnak** [riccardoperra/codeimage#640]
### Which @codeimage/* package(s) are relevant/releated to the feature request?

config

### Description

This app would be great for generating images for CLIs, but they often require custom colour styles.
Being able to configure colour would fix the issue.

**Comment by riccardoperra** [comment]
Hi, what do you mean about custom color style? It’s not clear

**Comment by juraj-hrivnak** [comment]
For example, I created this image from a screenshot from my terminal:
![](https://juraj-hrivnak.github.io/Pakku/juraj-hrivnak/Pakku/screenshot_ls.png)

As you can see different texts have different colours. With this suggestion, I mean that you could be able to select text and set its colour individually.

**Comment by riccardoperra** [comment]
I don't think this will be supported in the near future, CodeImage relies on codemirror6 and themes/colors are statically defined.

Customizable text decorations (color, bold, italic) needs a "text editor ui" which currently isn't in my plan
